pkg/fanal/image/image.go

```diff
@@ -116,7 +116,8 @@ func GuessBaseImageIndex(histories []v1.History) int {
 	var entrypointIndexFound bool
 	baseImageIndex := -1
 	var foundNonEmpty bool
-	for i := len(histories) - 1; i >= 0; i-- {
+	n := len(histories)
```

Comment:
Would it make sense to rename it so that its easier to understand. May be imageHistoryLength.